server/v1/controllers/auth.controller.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable arrow-parens */
+/* eslint-disable consistent-return */
+/* eslint-disable prefer-const */
+/* eslint-disable comma-dangle */
+import users from '../models/user.db';
+import user from '../models/user.model';
+import validate from '../helpers/user.validation';
+import auth from '../helpers/authenticate';
+
+class userController {
+  static signUp(req, res) {
+    let { error } = validate(user(req));
```

Comment:
Is there any reason why you did not use const here?

Reply:
Nope, I used it from my mind but i will change!